build/nuke/Native/Angle.cs

```diff
@@ -101,25 +101,21 @@ partial class Build {
                         // create universal mac binaries
                         var arm64NativeRuntimes = runtimes / "osx-arm64" / "native";
```

Comment:
We can also drop these variables I guess (and the associated EnsureCleanDirectory calls)

Reply:
Yeah... to be honest this was literally just because I couldn't be bothered to delete the files using GitHub UI haha. Will remove now.